client/js/render.js

```diff
@@ -173,6 +176,8 @@ function renderChannelUsers(data) {
 	if (search.val().length) {
 		search.trigger("input");
 	}
+
+	return users;
```

Comment:
This appears to be unnecessary?

Reply:
Look at R123, I need it to keep a reference on the user list that was just created.

Reply:
Oh my bad, the diff was rendered in such a way I assumed the return was for a wrong function.

Reply:
Hey, np, that just shows my change is not as easy to read as I'd hoped :) Do you want me to add a comment for this?

Reply:
Nah it's fine.